packages/matchbox/src/components/Tag/Tag.module.scss

```diff
@@ -114,14 +114,24 @@
 }
 
 .Content {
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-between;
   vertical-align: middle;
```

Comment:
Nice this works.

Couple questions - Do we still need `vertical-align: middle`? Looks like align-items takes precedence. And do we need `space-between` when the `.Tag` container is an inline element with no set width?

Reply:
Good questions. Let me play with it some more.